### ISSUE TITLE
Feature Request: improvement about  `prompts::DateSelect` for user friendly purpose

### ISSUE THREAD

**Issue by polarove** [mikaelmello/inquire#251]
## Is your feature request related to a problem? Please describe.

No. It's about localization and visual improvements

### Localization
Can we have localized title and days?
eg: from (English)
```sh
## prompt
? pick a date
>     august 2021
> mo tu we th fr sa su

## output
> pick a date August 15, 2021
```

to:  (French)
```sh
? Choisissez une date
>                   Août 2021
> Lundi Mardi Mercredi Jeudi Vendredi Samedi Dimanche

## output
> Choisissez une date 15 août 2021
```

Or any other languages.

### Visual improvements

Though we are able to set the range for dates to pick, those unselectable dates are too **visually clear** to be distinguished with selectable date.

## Describe alternatives you've considered

For localize days, pass days as parameters into a date enum with a default value accroding to itself
```rust
enum Days {
        Sunday(String),
        Monday(String),
        Tuesday(Input),
        Wednesday(String),
        Thursday(String),
        Friday(String),
        Saturday(String),
}
```

For title, I don't know either, perhaps this relates to the crate `chrono`.

**Comment by mikaelmello** [comment]
Absolutely, I'd love to have that. I'll spend some time and see if there's something ootb that does that, otherwise build an API for it.
